sdk/identity/azure-identity/azure/identity/_internal/aad_client.py

```diff
@@ -43,7 +43,7 @@ def obtain_token_by_client_secret(self, scopes: Iterable[str], secret: str, **kw
         return self._run_pipeline(request, **kwargs)
 
     def obtain_token_by_jwt_assertion(self, scopes: Iterable[str], assertion: str, **kwargs: Any) -> AccessToken:
-        request = self._get_jwt_assertion_request(scopes, assertion)
+        request = self._get_jwt_assertion_request(scopes, assertion, **kwargs)
```

Comment:
I don't get it.

Reply:
Do we need https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/azure/identity/aio/_internal/aad_client.py#L51 too?